realm/realm-library/src/syncIntegrationTest/java/io/realm/SyncedRealmIntegrationTests.java

```diff
@@ -324,15 +324,15 @@ public void waitForInitialRemoteData_readOnlyFalse_upgradeSchema() {
 
     @Test
     public void defaultRealm() throws InterruptedException {
-        SyncCredentials credentials = SyncCredentials.nickname("test", true);
+        SyncCredentials credentials = SyncCredentials.nickname("test", false);
         SyncUser user = SyncUser.logIn(credentials, Constants.AUTH_URL);
         SyncConfiguration config = SyncConfiguration.automatic();
         Realm realm = Realm.getInstance(config);
         SyncManager.getSession(config).downloadAllServerChanges();
         realm.refresh();
 
         try {
-            assertFalse(realm.isEmpty());
+            assertTrue(realm.isEmpty());
```

Comment:
FYI the `default` Realm is not completely empty on ROS, it has a default table `_DefaultRealmVersion` which maybe synchronised if you're not using partial Sync